ansible/roles/common/tasks/main.yml

```diff
@@ -39,6 +52,16 @@
   retries: 10
   delay: 15
   until: yumresult is not failed
+- name: "enable SELinux (Red Hat)"
+  block:
+    - name: "check if container-selinux is installed"
+      shell: "rpm -q --quiet container-selinux"
+      register: container_selinux_installed
+      failed_when: container_selinux_installed.rc == 2
+    - name: "install container-selinux"
```

Comment:
Why is this needed?

Reply:
When trying to install docker it was complaining about not having container-selinux. From the following blogpost, docker is not officially supported on RHEL 7:

https://nickjanetakis.com/blog/docker-tip-39-installing-docker-ce-on-redhat-rhel-7x